src/org/gk/database/AttributeTableModel.java

```diff
@@ -21,7 +21,7 @@ public Class getColumnClass(int col) {
     
     /**
      * Get the SchemaClass object for this AttributeTableModel.
-     * @return
+     * @return SchemaClass object
```

Comment:
Could this comment be made slightly more useful? Replace: "object" with "The SchemaClass object for this AttributeTableModel." Yes, I know it's copied-and-pasted from the method description, but the word "object" on its own is not terribly useful, the javadoc might as well not exist (and in this case, I think that could actually be forgiven ;) ).

Reply:
Fair point - I described the purpose of the SchemaClass object a little bit more in an additional commit.